_data/projects/nativescript.yml

```diff
@@ -0,0 +1,10 @@
+name: NativeScript
+desc: Cross platform Native mobile apps with JavaScript
+site: https://nativescript.org
+tags:
+- JavaScript
+- mobile
+- native
+upforgrabs:
+  name: PRs plz!
+  link: https://github.com/nativescript/nativescript/labels/up-for-grabs
```

Comment:
This URL should point to an existing tag on the project - unfortunately `up-for-grabs` doesn't exist on their repo.

Perhaps you could ask the team if this is something they're interested in...